forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifierTest.java

```diff
@@ -132,8 +132,7 @@ public void shouldFailToVerifyB64HeaderGivenVersionBefore3_1_4AndB64HeaderIsNonB
         OBErrorException exception = catchThrowableOfType(() -> detachedJwsVerifier.verifyDetachedJws(detachedJws, OBVersion.v3_1_3, request, principal), OBErrorException.class);
 
         // Then
-        assertThat(exception).hasMessage("Invalid detached signature " + detachedJws + ". Reason: b64 claim header " +
-                "not set to false");
+        assertThat(exception).hasMessage("Internal Server Error");
```

Comment:
I don't like this error message

Reply:
Could you have this within verifyDetachedJws():
```
catch (ParseException e) {
   log.error("Can't parse JWS", e);
   throw new OBErrorException(OBRIErrorType.DETACHED_JWS_INVALID, detachedJws, e.getMessage());
}
```

Reply:
That's much better